app/models/miq_server.rb

```diff
@@ -625,7 +625,7 @@ def display_name
   end
 
   def server_timezone
-    get_config("vmdb").config.fetch_path(:server, :timezone) || "UTC"
+    @timezone ||= get_config("vmdb").config.fetch_path(:server, :timezone) || "UTC"
```

Comment:
We have `my_server(true)` (28 instances). So I thought it would get cleared frequently enough.